packages/longhorn-crd/longhorn-1.4/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,40 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -1,28 +1,11 @@
++annotations:
++  catalog.cattle.io/certified: rancher
++  catalog.cattle.io/hidden: "true"
++  catalog.cattle.io/namespace: longhorn-system
++  catalog.cattle.io/release-name: longhorn-crd
+ apiVersion: v1
+ appVersion: v1.4.0
+-description: Longhorn is a distributed block storage system for Kubernetes.
+-home: https://github.com/longhorn/longhorn
+-icon: https://raw.githubusercontent.com/cncf/artwork/master/projects/longhorn/icon/color/longhorn-icon-color.png
+-keywords:
+-- longhorn
+-- storage
+-- distributed
+-- block
+-- device
+-- iscsi
+-- nfs
+-kubeVersion: '>=1.21.0-0'
+-maintainers:
+-- email: maintainers@longhorn.io
+-  name: Longhorn maintainers
+-name: longhorn
+-sources:
+-- https://github.com/longhorn/longhorn
+-- https://github.com/longhorn/longhorn-engine
+-- https://github.com/longhorn/longhorn-instance-manager
+-- https://github.com/longhorn/longhorn-share-manager
+-- https://github.com/longhorn/longhorn-manager
+-- https://github.com/longhorn/longhorn-ui
+-- https://github.com/longhorn/longhorn-tests
+-- https://github.com/longhorn/backing-image-manager
+-version: 1.4.0
++description: Installs the CRDs for longhorn.
++name: longhorn-crd
++type: application
++version: 100.3.0+up1.4.0
```

Comment:
Comparing to the old versions, this line should be just `version: 1.4.0` because this one is the application version, not the chart version

Reply:
Got it, I've solved it. Thank you very much.